tornjak-frontend/src/components/spiffe-helper.js

```diff
@@ -88,6 +88,8 @@ class SpiffeHelper extends Component {
   // the list of entries which are associated with this agent
   getAgentEntries(agent, entries) {
     let nodeEntries = entries.filter(e => e.parent_id.path === "/spire/server")
+    if(agent === undefined || agent[0].selectors === undefined) 
```

Comment:
this is probably not the right fix, the result of selectors being defined probably because the agent doesnt have selectors.... so in this case, the way to treat it is to treat undefined selectors as the empty set (or list).

Reply:
taken care of!